### PR TITLE
fix(opex-config): [CHK-1723] Add configuration for checkout opex dashboards

### PR DESCRIPTION
**Pull request by pietro-tota**
#### List of Changes
- Restore openapi url to checkout infra api
- Add per endpoint configuration override 

#### Motivation and Context
- Restore openapi url to checkout infra api
- Add per endpoint configuration override 

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .opex/ecommerce-checkout/env/prod/config.yaml [hunk 1]
```diff
@@ -10,4 +10,104 @@ action_groups:
 overrides:
   hosts: # Use these hosts instead of those inside the OpenApi spec
     - api.platform.pagopa.it
-  base_path: "ecommerce/checkout/v1"
\ No newline at end of file
+  base_path: "ecommerce/checkout/v1"
+  endpoints:
+    /payment-requests/{rpt_id}:
+      availability_threshold: 0.95 # Default: 99%
```
Comment (pietro-tota) [inline .opex/ecommerce-checkout/env/prod/config.yaml]: TODO: customize those value with wanted availability thresholds and so on

File: .opex/ecommerce-checkout/env/prod/config.yaml [hunk 2]
```diff
@@ -10,4 +10,104 @@ action_groups:
...
+    /transactions:
+      availability_threshold: 0.95 # Default: 99%
+      availability_evaluation_frequency: 30 # Default: 10
+      availability_evaluation_time_window: 50 # Default: 20
+      availability_event_occurrences: 3 # Default: 1
+      response_time_threshold: 2 # Default: 1
+      response_time_evaluation_frequency: 35 # Default: 10
+      response_time_evaluation_time_window: 55 # Default: 20
+      response_time_event_occurrences: 5 # Default: 1
+    /transactions/{transactionId}:
```
Comment (ciuffagianluca) [inline .opex/ecommerce-checkout/env/prod/config.yaml:33]: All urls are correctly listed here. 
Just a point, we have a POST and DELETE with this path `/transactions/{transactionId}`
According to you this values will override both apis?

Comment (pietro-tota) [reply .opex/ecommerce-checkout/env/prod/config.yaml:33]: Yes. 
Alerts generated by opex take into account all requests made to openapi paths with no discrimination against http method.

**Review by infantesimone — APPROVED** [review]

**Review by ciuffagianluca — APPROVED** [review]
🧇 good!

**Pull request merged**